server/config/topic_page/equity/USA_states.textproto

```diff
@@ -0,0 +1,1215 @@
+    description: "Percentage of the population of each sex living below the poverty level."
+    left_tiles {
+      type: MAP
+      title: "Percentage of the female population in poverty (${date})"
+      stat_var_key: "poverty_rate_female"
+    }
+    right_tiles {
+      type: MAP
+      title: "Percentage of the male population in poverty (${date})"
+      stat_var_key: "poverty_rate_male"
+    }
+    left_tiles {
+      type: LINE
+      title: "Poverty rate, by sex"
+      stat_var_key: "poverty_rate_female"
+      stat_var_key: "poverty_rate_male"
+    }
+    right_tiles {
+      type: RANKING
+      ranking_metadata {
+        show_highest: true
+      }
+      stat_var_key: "poverty_rate_female"
+      stat_var_key: "poverty_rate_male"
+    }
+  }
+
+  blocks {
+    title: "Uninsured, by sex"
+    description: "Of those without insurance, how many are male or female?"
+    # TODO(beets): This isn't available at the country level
+    # stat_var_metadata {
+    #   stat_var: "Count_Person_Female_WithHealthInsurance"
+    #   denom: "Count_Person_Female"
+    #   unit: "%"
+    #   scaling: 100
+    #   name: "Female, insured"
+    # }
+    # stat_var_metadata {
+    #   stat_var: "Count_Person_Male_WithHealthInsurance"
+    #   denom: "Count_Person_Male"
+    #   unit: "%"
+    #   scaling: 100
+    #   name: "Male, insured"
+    # }
+    right_tiles {
+      type: LINE
+      title: "Uninsured, by sex"
+      stat_var_key: "uninsured_rate_female"
+      stat_var_key: "uninsured_rate_male"
+    }
+    left_tiles {
+      type: HIGHLIGHT
+      description: "Americans have no health insurance (${date})"
+      stat_var_key: "uninsured_count"
+    }
+    left_tiles {
+      type: RANKING
+      ranking_metadata {
+        show_highest: true
+      }
+      stat_var_key: "uninsured_rate_female"
+      stat_var_key: "uninsured_rate_male"
+    }
+  }
+
+  blocks {
+    title: "With disabilities, by sex"
+    description: "Percentage of the population of each sex with disabilities"
+    # TODO(beets): This isn"t available at the country level
+    # stat_var_metadata {
+    #   stat_var: "Count_Person_Female_WithHealthInsurance"
+    #   denom: "Count_Person_Female"
+    #   unit: "%"
+    #   scaling: 100
+    #   name: "Female, insured"
+    # }
+    # stat_var_metadata {
+    #   stat_var: "Count_Person_Male_WithHealthInsurance"
+    #   denom: "Count_Person_Male"
+    #   unit: "%"
+    #   scaling: 100
+    #   name: "Male, insured"
+    # }
+    left_tiles {
+      type: LINE
+      title: "With disability, by sex in ${place}"
+      stat_var_key: "disability_rate_female"
+      stat_var_key: "disability_rate_male"
+    }
+    right_tiles {
+      type: RANKING
+      ranking_metadata {
+        show_highest: true
+      }
+      stat_var_key: "disability_rate_female"
+      stat_var_key: "disability_rate_male"
+    }
+  }
+
+  # Note: This data is not available by race.
+  blocks {
+    title: "Education attainment by sex"
```

Comment:
nit: for consistency with all other titles, maybe "Education attainment, by sex"

Reply:
thanks for the catch, fixed this elsewhere too